app/components/settings/index.js

```diff
@@ -0,0 +1,182 @@
+          contentContainerStyle={styles.listContent}
+          showsVerticalScrollIndicator={false}
+          showsHorizontalScrollIndicator={false}
+        >
+          <Text style={styles.label}>
+            {I18n.t('settings.loggedIn')}
+          </Text>
+
+          <View style={styles.user}>
+            <View style={styles.info}>
+              <Text style={styles.name}>
+                {this.props.user.fullName}
+              </Text>
+              <Text style={styles.email}>
+                {this.props.user.email}
+              </Text>
+            </View>
+            <TouchableHighlight
+              activeOpacity={0.5}
+              underlayColor="transparent"
+            >
+              <Text style={styles.logout}>LOG OUT</Text>
+            </TouchableHighlight>
+          </View>
+
+          {areas && areas.length
+            ? <View style={styles.areas}>
+              <Text style={styles.label}>
+                {I18n.t('settings.yourAreas')}
+              </Text>
+
+              {areas.map((item, key) => {
+                const area = item.attributes;
+                area.id = item.id;
+                const image = areasImages[area.id];
+
+                return (
+
+                  <TouchableHighlight
+                    key={key}
+                    activeOpacity={0.5}
+                    underlayColor="transparent"
+                  >
+                    <View style={styles.item}>
+                      <View style={styles.imageContainer}>
+                        {image
+                          ? <Image style={styles.image} source={{ uri: image }} />
+                          : null
+                        }
+                      </View>
+                      <Text style={styles.title} numberOfLines={2}>
+                        {area.name}
+                      </Text>
+                      <TouchableHighlight
+                        activeOpacity={0.5}
+                        underlayColor="transparent"
+                      >
+                        <Image style={Theme.icon} source={nextIcon} />
+                      </TouchableHighlight>
+                    </View>
+                  </TouchableHighlight>
+                );
+              })
+              }
+            </View>
+          : null
+          }
+          <View style={styles.addButton}>
+            <TouchableHighlight
+              activeOpacity={0.5}
+              underlayColor="transparent"
+            >
+              <Image style={Theme.icon} source={plusIcon} />
+            </TouchableHighlight>
+            <Text style={styles.addButtonText}>
+              {I18n.t('settings.addArea').toUpperCase()}
+            </Text>
+          </View>
+
+          <View style={styles.aboutSection}>
+            <Text style={styles.label}>
+              {I18n.t('settings.aboutApp')}
+            </Text>
+
+            <View style={styles.aboutList}>
+              {aboutSections.map((item, key) => {
```

Comment:
Could be useful to make this list a component to reuse later in other screens and parametrize it.